xml/containers-basics.xml

```diff
@@ -135,21 +134,18 @@
   <para>
    Both base and custom images are usually available through a repository of
    images called registry. Unless a registry is explicitly specified, &podman;
-   pulls images from the Docker Hub registry.
+   pulls images from the &dhub; registry. While you can fetch a base image manually, &podman; can do that automatically when building a custom image.
   </para>
 
   <para>
-   After you pull a base image, you can use it as a starting point for building
-   a custom image. For example, you can use the SLE base image to generate an
-   image that is configured to run a specific application. To build a custom
-   image, you need to create a special file called Dockerfile, containing
-   building instructions. For example, a Dockerfile can contain instructions to
-   update the system software, install the desired application, open specific
-   network ports, run commands, etc.
+   To build a custom image, you need to create a special file called Dockerfile,
```

Comment:
Do we not also document how to build containers with our KIWI stack?

Reply:
Yes, eventually. Podman and Buildah are the first priority.